ajax/libs/vue2-leaflet/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "vue2-leaflet",
+  "filename": "vue2-leaflet.min.js",
+  "version": "1.0.1",
+  "description": "Vue2 leaflet library",
+  "author": "Mickael Bouchaud",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/KoRiGaN/Vue2Leaflet.git"
+  },
+  "keywords": [
+    "vue",
+    "leaflet",
+    "map",
+    "vuejs"
+  ],
+  "homepage": "https://korigan.github.io/Vue2Leaflet/#",
```

Comment:
Let's remove it, refer https://github.com/cdnjs/cdnjs/pull/10729#discussion_r104134423
> The homepage should at least have download or repo links.